test/e2e/utils/httpClient.go

```diff
@@ -85,16 +82,13 @@ func (c *httpClientConfig) Post(path, payload string, headers map[string]string)
 
 func (c *httpClientConfig) exec(req *http.Request, headers map[string]string) (*http.Response, error) {
 	if c.isAuthRequired() {
-		if c.authRealm == nil {
-			rsp, err := c.Do(req)
-			ExpectNoError(err)
-			if rsp.StatusCode != http.StatusUnauthorized {
-				return rsp, fmt.Errorf("Expected 401 DIGEST response before content: %v", rsp)
-			}
-			c.authRealm = getAuthorization(*c.username, *c.password, rsp)
+		rsp, err := c.Do(req)
```

Comment:
@ryanemerson probably this situation could happens when later `rsp.Body.Close()` is not calling